v2api/create_socrata_csv.py

```diff
@@ -26,6 +26,10 @@
 import pandas as pd
 import requests
 from .query_v2_api import get_filer, get_auth_from_env_file
+# <<<< dedupe script <<<<
+from .misfiled_ammendments_dedupe import main, is_iso_str_in_range
+filing_date = main()
```

Comment:
ah, sorry, didn't notice before you were importing this. the pattern I generally see is to use `main()` only for what would happen you ran the file as a script, like
```python
if __name__ == '__main__':
    main()
```
in this case, since it's something you're importing, I'd give it a more meaningful name.